contracts/axone-objectarium/Cargo.toml

```diff
@@ -43,9 +42,11 @@ thiserror.workspace = true
 
 [dev-dependencies]
 base64 = "0.22.1"
-cw-multi-test.workspace = true
 testing.workspace = true
 
 [features]
 # use library feature to disable all instantiate/execute/query exports
 library = []
+
+[package.metadata.cargo-machete]
+ignored = ["md5"]
```

Comment:
I agree. But it seems that `md5` is incorrectly flagged as unused (a false positive) by `machete`. To prevent machete from removing it, we should add `md5` to its ignore list.